cmd/telliot/cmds.go

```diff
@@ -470,84 +544,114 @@ func (m mineCmd) Run() error {
 
 	// Not using a remote DB so need to start the trackers.
 	if cfg.Mine.RemoteDBHost == "" {
-		ds, err = ops.CreateDataServerOps(ctx, logger, cfg, proxy, client, contract, account, ch1)
+		err = config.ValidateDataServerConfig(cfg)
+		if err != nil {
+			return errors.Wrapf(err, "validating config")
+		}
```

Comment:
Here we extract the pub keys from the private keys since in mining mode we require private keys.